### PR TITLE
[com_menus] Hide move/copy in admin menu batch modal

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #15133

### Steps to reproduce the issue
Create an Administrator menu.
Create a menu item under this Administrator menu.
Click this menu item.
Click `Batch` button.

### Expected result
Copy/Move radios to display only after a menu/parent item is selected.

### Actual result
batchmenu

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on a47d851844c537622d63368902e0411bf1fbe4a8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18216">issues.joomla.org/tracker/joomla-cms/18216</a>.</sub>

File: administrator/components/com_menus/views/items/tmpl/default_batch_body.php [hunk 1]
```diff
@@ -15,6 +15,26 @@
...
+							$("#batch-copy-move").show();
+						} else {
+							$("#batch-copy-move").hide();
+						}
+					});
+				}
+			});
+				'
+	);
+	endif;
```
Comment (Quy) [inline administrator/components/com_menus/views/items/tmpl/default_batch_body.php]: Remove tab.

File: administrator/components/com_menus/views/items/tmpl/default_batch_body.php [hunk 2]
```diff
@@ -15,6 +15,26 @@
...
+					batchSelector.on("change", function(){
+						if (batchSelector.val() != 0 || batchSelector.val() != "") {
+							$("#batch-copy-move").show();
+						} else {
+							$("#batch-copy-move").hide();
+						}
+					});
+				}
+			});
+				'
```
Comment (Quy) [inline administrator/components/com_menus/views/items/tmpl/default_batch_body.php]: Remove 2 tabs.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 47af3982a3695154ee315fc3b3dee7e0dc4846ef<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18216">issues.joomla.org/tracker/joomla-cms/18216</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 47af3982a3695154ee315fc3b3dee7e0dc4846ef<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/18216">issues.joomla.org/tracker/joomla-cms/18216</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**